broker-core/src/main/java/org/apache/qpid/server/store/BrokerStoreUpgraderAndRecoverer.java

```diff
@@ -805,11 +805,10 @@ private static class VirtualHostEntryUpgrader
                     addAttributeTransformer("jdbcBytesForBlob", addContextVar("qpid.jdbcstore.useBytesForBlob")).
                     addAttributeTransformer("jdbcBlobType", addContextVar("qpid.jdbcstore.blobType")).
                     addAttributeTransformer("jdbcVarbinaryType", addContextVar("qpid.jdbcstore.varBinaryType")).
-                    addAttributeTransformer("partitionCount", addContextVar("qpid.jdbcstore.bonecp.partitionCount")).
-                    addAttributeTransformer("maxConnectionsPerPartition",
-                                            addContextVar("qpid.jdbcstore.bonecp.maxConnectionsPerPartition")).
-                    addAttributeTransformer("minConnectionsPerPartition",
-                                            addContextVar("qpid.jdbcstore.bonecp.minConnectionsPerPartition")),
+                    addAttributeTransformer("maximumPoolSize",
+                                            addContextVar("qpid.jdbcstore.hikaricp.maximumPoolSize")).
+                    addAttributeTransformer("minimumIdle",
+                                            addContextVar("qpid.jdbcstore.hikaricp.minimumIdle")),
```

Comment:
Hi Robbie,

Thank you for the review. You're right, that needs to be fixed. 

Are there any conventions regarding the model version change? It seems to me that as only the context attribute names / values were changed without adding new entities or attributes, model version could be changed from 9.0 to 9.1? Or should the model version match the broker version as well? E.g. model version 9.2 for broker version 9.2.0 until other model change or if model changes to 10 then qpid-broker-j version have to be changed 10.0.0 as well?

Reply:
I'm not 100% sure, its a long time since I really contributed to the broker and things changed since then. Originally when we added that, I believe it was just its own independent config version. From broker-core/src/main/java/org/apache/qpid/server/model/BrokerModel.java it looks like around the 6.0.0 release (when the release version was bumped as everything became an independent component, rather than the big 'qpid release' with everything) that the model version was also jumped to bring it into 'major 6, minor 0' alignment, and since then it seems like it has consistently had changes that mean the major matched the broker release major version. It looks like maybe the minors too. Perhaps digging more at the changes that have been made historically will make it clearer for you what would be best, but using either 9.1.0 (next minor) or 9.2.0 (matching minor) seems fair if the release version is 9.2.0.